### ISSUE TITLE
configure cannot build out-of-tree

### ISSUE THREAD

**Issue by mirabilos** [emersion/mrsh#152]
mkdir build
    cd build
    ../mrsh/configure

… fails with…

    Unable to specify exported symbols (is Linux supported?)

Trying to debug the hard-to-debug (comparatively) script, it turns out it expects the `libmrsh.gnu.sym` file in the cwd.

**Comment by emersion** [comment]
Even if the `configure` script worked, `make` would fail (because the Makefile is still in the source tree, it doesn't get copied over). Are out-of-tree builds something that is in general supported by make-based build systems?

**Comment by mirabilos** [comment]
Yes, e.g. GNU autoconf copies the Makefile (with substitutions) and everything else needed, and in general just points to the source directory.

**Comment by emersion** [comment]
I'm not talking about autotools, I'm talking about other software using raw Makefiles like we do.

**Comment by mirabilos** [comment]
Simon Ser dixit:

>I'm not talking about autotools, I'm talking about other software using
>raw Makefiles like we do.

It’s not about the Makefile, it’s about the configure script.
